### PR TITLE
Prep for v1.18.0

### PR DESCRIPTION
**Pull request by odow**
## Basic

 - [x] `version` field of `Project.toml` has been updated
       - If a breaking change, increment the MAJOR field and reset others to 0
       - If adding new features, increment the MINOR field and reset PATCH to 0
       - If adding bug fixes or documentation changes, increment the PATCH field

## Documentation

 - [x] Add a new entry to `docs/src/changelog.md`, following existing style

## Tests

 - [x] The `solver-tests.yml` GitHub action does not have unexpected failures.
         https://github.com/jump-dev/MathOptInterface.jl/actions/runs/5340532776

### CODE REVIEW

File: docs/src/changelog.md [hunk 1]
```diff
@@ -7,6 +7,27 @@ CurrentModule = MathOptInterface
...
+ - Fixed a missing `@require` in `MOI.Test` (#2195) (#2196)
+ - Fixed incorrect usage of `Utilities.operate!` in bridges (#2207)
+
+### Other
+
+ - Removed unused argument names (#2199)
+ - Reduced memory requirement for tests (#2204)
+ - Refactored `Utilities.promote_operation` (#2206)
+ - Improved code style in `Utilities/mutable_arithmetics.jl` (#2209)
+ - Refactored `operate_` methods in `Utilities` (#2208) (#2212) (#2214) (#2215)
```
Comment (odow) [inline docs/src/changelog.md:29]:
```suggestion
 - Refactored various methods in `Utilities/functions.jl` (#2208) (#2212)
   (#2213) (#2214) (#2215)
```

File: docs/src/changelog.md [hunk 2]
```diff
@@ -7,6 +7,28 @@ CurrentModule = MathOptInterface
...
+### Added
+
+ - Added `Bridges.Objective.SlackBridgePrimalDualStart` (#2194)
+ - Added [`MOI.constraint_gradient_structure`](@ref) and
+   [`MOI.eval_constraint_gradient`](@ref) (#2200)
+
+### Fixed
+
+ - Fixed a missing `@require` in `MOI.Test` (#2195) (#2196)
+ - Fixed incorrect usage of `Utilities.operate!` in bridges (#2207)
```
Comment (odow) [inline docs/src/changelog.md:21]:
```suggestion
 - Fixed incorrect usage of `Utilities.operate!` in bridges (#2207) (#2216)
```

**Comment by odow** [comment]
@blegat PolyJuMP and SumOfSquares can't install?

**Comment by blegat** [comment]
Yes, that's to be expected, they are using the master branch of some dependencies (hoping to release them today)

**Review by blegat — APPROVED** [review]

**Comment by odow** [comment]
Wait for https://github.com/jump-dev/MathOptInterface.jl/pull/2221

File: docs/src/changelog.md [hunk 3]
```diff
@@ -7,6 +7,28 @@ CurrentModule = MathOptInterface
...
+### Added
+
+ - Added `Bridges.Objective.SlackBridgePrimalDualStart` (#2194)
+ - Added [`constraint_gradient_structure`](@ref) and
+   [`eval_constraint_gradient`](@ref) (#2200)
+
+### Fixed
+
+ - Fixed a missing `@require` in `MOI.Test` (#2195) (#2196)
+ - Fixed incorrect usage of `Utilities.operate!` in bridges (#2207) (#2216)
```
Comment (odow) [inline docs/src/changelog.md:21]:
```suggestion
 - Fixed incorrect usage of `Utilities.operate!` in bridges (#2207) (#2216)
 - Fixed splatting nonlinear expression with univariate operator (#2221)
```

**Pull request merged**